packages/vhd-format-lwt/vhd-format-lwt.0.12.3/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: ["Dave Scott" "Jon Ludlam"]
+tags: ["org:mirage" "org:xapi-project"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/mirage/ocaml-vhd"
+doc: "https://mirage.github.io/ocaml-vhd/"
+bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "cstruct"
```

Comment:
```suggestion
  "cstruct" {< "6.1.0"}
```
Due to the `Unbound value Cstruct.len` failure

Reply:
Even though this version of vhd-format has some bounds, in some tests, due to the freedom in picking any version of `vhd-format`, there is the failure appearing